core/data_migration_scripts/4_6_0_stages/EE_DMS_4_6_0_country_system_question.dmsstage.php

```diff
@@ -32,26 +32,26 @@ public function __construct()
 
     /**
      * updates the question with the new question type
-     * @param array $question an associative array where keys are column names and values are their values.
+     * @param array $event_question_group an associative array where keys are column names and values are their values.
      * @return null
      */
-    protected function _migrate_old_row($question)
+    protected function _migrate_old_row($event_question_group)
```

Comment:
oups this shouldn't have been renamed. But I don't think this would have introduced any bugs